superset/db_engine_specs.py

```diff
@@ -973,28 +973,37 @@ def fetch_data(cls, cursor, limit):
     @staticmethod
     def create_table_from_csv(form, table):
         """Uploads a csv file and creates a superset datasource in Hive."""
-        def get_column_names(filepath):
-            with open(filepath, 'rb') as f:
-                return next(unicodecsv.reader(f, encoding='utf-8-sig'))
-
-        table_name = form.name.data
-        filename = form.csv_file.data.filename
-
-        bucket_path = app.config['CSV_TO_HIVE_UPLOAD_S3_BUCKET']
+        def convert_to_hive_type(col_type):
+            """maps tableschema's types to hive types"""
+            TABLESCHEMA_TO_HIVE_TYPES = {
```

Comment:
Sorry given this no longer resides in the config this should be a lower case variable name.